### PR TITLE
Improves the english a bit in the readme

### PR DESCRIPTION
**Pull request by peteratticusberg**
Some edits here fix grammatical errors while others are just to make the english sound more idiomatic.

Thanks for all your work on this! Love the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![textlint logo](http://textlint.github.io/media/logo/spaced/textlint-logo.png)](http://textlint.github.io/)
 
 > The pluggable linting tool for text and markdown.  
-> It is similar to [ESLint](http://eslint.org/ "ESLint"), but textlint for natural language.
+> TextLint is similar to [ESLint](http://eslint.org/ "ESLint"), but it's for use with natural language.
```
Comment (azu) [inline README.md]: TextLint => textlint

File: README.md [hunk 2]
```diff
@@ -14,26 +14,24 @@
 
 ## Online Demo
 
-Visit [https://textlint.github.io/](https://textlint.github.io/) and type text!
+Visit [https://textlint.github.io/](https://textlint.github.io/) to see TextLint in action!
```
Comment (azu) [inline README.md]: TextLint => textlint

File: README.md [hunk 3]
```diff
@@ -44,18 +42,18 @@ $ npm install textlint -g
...
 
-Test: Run `node -v` in your console. The version should be higher than v4.0.0.
+If you're not sure what version of Node you're running, you can run `node -v` in your console to find out. TextLint works on versions v4.0.0 and up.
 
-:warning: Caution: Mixed location of installation.
+:warning: Note:
 
-- If you have installed `textlint` as `--global`(`-g`), must install each rule as `--global`.
-- If you have installed `textlint` as `--save-dev`(`-D`), must install each rule as `--save-dev`.
+- If you have installed `textlint` globally (e.g. `npm install textlint --global`), you must install each rule globally as well `--global`.
```
Comment (azu) [inline README.md]: https://ci.appveyor.com/project/azu/textlint/build/1.0.865#L491
[rousseau](https://github.com/GitbookIO/rousseau) warn "readbility".

The sentence may be too long.

File: README.md [hunk 4]
```diff
@@ -178,9 +169,9 @@ $ textlint README.md
 }
 ```
 
-Pass rule's options ("key": "value") to `very-nice-rule`.
+For example here we pass the options ("key": "value") to `very-nice-rule`.
 
-It mean that use the following format:
+Options can be specified in your .textlintrc file as follows:
```
Comment (azu) [inline README.md]: Wrap `.textlintrc` with backtick

**Comment by azu** [comment]
@peteratticusberg Thanks!

**Comment by peteratticusberg** [comment]
Anytime!

**Pull request merged**